tests/alternatives/main.fmf

```diff
@@ -0,0 +1,14 @@
+summary: Alternatives integration tests
+component: alternatives
+framework: beakerlib
+contact: Jan Macku <jamacku@redhat.com>
+tier: 1
+require:
+  - gcc
```

Comment:
nit: you may want to sort this list

Reply:
I actually like the long list a bit more in this case.